modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/EpicsUtil.scala

```diff
@@ -42,6 +42,10 @@ object EpicsCommand {
     case \/-(e) => e
   })
 
+  def setParameter[A,T](p: Option[CaParameter[T]], v: A, f: A => T): SeqAction[Unit] =
+    safe(EitherT(Task.delay {
```

Comment:
`safe` is defined in the same file. It transforms a SeqAction to capture exceptions and put them in the Task output as a left value.